### PR TITLE
ENH: Add Neurostar Support in README

### PR DESCRIPTION
**Pull request by GuillaumeTh**
Add the Neurostar link and tag in the README.

### CODE REVIEW

**Review by arnaudbore — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ This documentation presents how to install and launch TractoFlow on a local comp
 
 If you are a user and NOT A DEVELOPER, we HIGHLY RECOMMEND following the instructions on the documentation website.
 
+
+Support:
+--------
+
+For questions or help on TractoFlow, please create a topic on Neurostar with the `tractoflow` tag [https://neurostars.org](https://neurostars.org)
+
+Before posting, please check the topics already created. [https://neurostars.org/tags/tractoflow](https://neurostars.org/tags/tractoflow)
```
Comment (arnaudbore) [inline README.md]: are already or has already been

File: README.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ This documentation presents how to install and launch TractoFlow on a local comp
 
 If you are a user and NOT A DEVELOPER, we HIGHLY RECOMMEND following the instructions on the documentation website.
 
+
+Support:
+--------
+
+For questions or help on TractoFlow, please create a topic on Neurostar with the `tractoflow` tag [https://neurostars.org](https://neurostars.org)
```
Comment (arnaudbore) [inline README.md]: .

File: README.md [hunk 3]
```diff
@@ -16,6 +16,15 @@ This documentation presents how to install and launch TractoFlow on a local comp
 
 If you are a user and NOT A DEVELOPER, we HIGHLY RECOMMEND following the instructions on the documentation website.
 
+
+Support:
+--------
+
+For questions or help on TractoFlow, please create a topic on Neurostar with the `tractoflow` tag. [https://neurostars.org](https://neurostars.org)
```
Comment (jchoude) [inline README.md]: You should create the link directly for the `Neurostar` word (in `on Neurostars`). Also, missing a `s` for the first `Neurostar`.

File: README.md [hunk 4]
```diff
@@ -16,6 +16,15 @@ This documentation presents how to install and launch TractoFlow on a local comp
...
+
+Before posting, please check the topics already created. [https://neurostars.org/tags/tractoflow](https://neurostars.org/tags/tractoflow)
```
Comment (jchoude) [inline README.md]: I would put the link directly over `check the topics`.

**Review by jchoude — APPROVED** [review]

**Pull request merged**